src/commands/export.ts

```diff
@@ -417,12 +423,21 @@ command
         projectOwnerType,
         proxyUrl,
         repositoryMappingsOutputPath,
+        skipCertificateVerification,
         skipUpdateCheck,
         verbose,
       } = opts;
 
       const logger = createLogger(verbose);
 
+      if (skipCertificateVerification) process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+      if (proxyUrl && !skipCertificateVerification) {
```

Comment:
This maintains compatibility with the existing approach of setting the environment variable:

```suggestion
      if (proxyUrl && process.env.NODE_TLS_REJECT_UNAUTHORIZED !== '0') {
```